### PR TITLE
Accountingservice added restaurant & courier

### PR DESCRIPTION
**Pull request by shohrukh56**

### CODE REVIEW

File: app/services/accounting/api/v1/handlers/curiergrp/couriergrp.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+	}
+}
+
+func (c CourierHandler) GetCourierAccount(rw http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	id, ok := vars[CourierIDKey]
+	if !ok {
+		if err := accountingapi.Respond(rw, http.StatusBadRequest, errNoCourierIDParam); err != nil {
+			c.log.Println(err)
+		}
```
Comment (VolodymyrShchur) [inline app/services/accounting/api/v1/handlers/curiergrp/couriergrp.go:81]: return don't needed here?

File: app/services/accounting/api/v1/handlers/routs/routs.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package routs
```
Comment (VolodymyrShchur) [inline app/services/accounting/api/v1/handlers/routs/routs.go:1]: routes

File: app/services/accounting/pkg/service/courierservice/courierservice.go [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+		c.logger.Println(err)
+		return "", systemErr
+	}
+
+	return "courier account deleted", nil
+}
+
+func (c Service) AddToBalanceCourierAccount(account domain.CourierAccount) (*domain.CourierAccount, error) {
+	if account.CourierID < 1 {
+		return nil, errWrongCourierIDType
```
Comment (VolodymyrShchur) [inline app/services/accounting/pkg/service/courierservice/courierservice.go:113]: wrong type or value?

File: app/services/accounting/pkg/storage/consumerstorage/consumerstorage_psql.go [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+		db: p.DB,
+	}
+}
+
+func (c Storage) InsertNewConsumerAccount(consumerAccount domain.ConsumerAccount) (*domain.ConsumerAccount, error) {
+	sql := `INSERT INTO
+				consumer_account
+					(consumer_id, created_at, updated_at)
+			VALUES($1,now(),now())
+			returning *`
```
Comment (VolodymyrShchur) [inline app/services/accounting/pkg/storage/consumerstorage/consumerstorage_psql.go:28]: format

File: app/services/accounting/pkg/storage/consumerstorage/consumerstorage_psql.go [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+	}
+
+	return &newCourier, nil
+}
+
+func (c Storage) GetConsumerAccountByID(id uint64) (*domain.ConsumerAccount, error) {
+	sql := `SELECT * FROM 
+				consumer_account
+			WHERE
+				consumer_id = $1
```
Comment (VolodymyrShchur) [inline app/services/accounting/pkg/storage/consumerstorage/consumerstorage_psql.go:43]: change to more readable format

File: app/services/accounting/api/v1/handlers/routs/routs.go [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+		ConsumerService: app.ConsumerService,
+	})
+
+	app.ServeMux.HandleFunc(version+"/status", cgh.StatusHandler).Methods(http.MethodPost)
+
+	app.ServeMux.HandleFunc(version+"/consumers/account", cgh.InsertNewConsumerAccount).Methods(http.MethodPost)
+	app.ServeMux.HandleFunc(version+"/consumers/account/{"+consumergrp.ConsumerIDKey+"}", cgh.GetConsumerAccount).Methods(http.MethodGet)
+	app.ServeMux.HandleFunc(version+"/consumers/account/{"+consumergrp.ConsumerIDKey+"}", cgh.DeleteConsumerAccount).Methods(http.MethodDelete)
+
+	app.ServeMux.HandleFunc(version+"/consumers/account/balance-add", cgh.AddToBalanceConsumerAccount).Methods(http.MethodPut)
```
Comment (shohrukh56) [inline app/services/accounting/api/v1/handlers/routs/routs.go:32]: consumersaccount/{consumer_id}/addbalance

File: app/services/accounting/api/v1/handlers/consumergrp/consumergrp.go [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+			c.log.Println(err)
+		}
+		return
+	}
+
+	account := requestToNewConsumerAccount(&newConsumerAccountRequest)
+
+	data, err := c.service.InsertNewConsumerAccount(account)
+
+	if err != nil && err == systemErr {
```
Comment (shohrukh56) [inline app/services/accounting/api/v1/handlers/consumergrp/consumergrp.go:83]: try use errors is /as

**Pull request merged**